lib/atlas/sparse_graph_query.rb

```diff
@@ -0,0 +1,41 @@
+module Atlas
+  class SparseGraphQuery
+    include ActiveDocument
+
+    FILE_SUFFIX = 'gql'
```

Comment:
Do you prefer `.gql` for this? Personally I'd quite like to someday rename all the gqueries to use `.ad` and get rid of the `.gql` extension; the syntax of the files are identical after all.